packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx

```diff
@@ -111,6 +132,12 @@ export class FacepileBase extends BaseComponent<IFacepileProps, {}> {
 
   private _getPersonaControl(persona: IFacepilePersona): JSX.Element {
     const { getPersonaProps, personaSize } = this.props;
+    const personaStyles = (props: IPersonaStyleProps): Partial<IPersonaStyles> => ({
```

Comment:
You can pass in an object, rather than a function, and it can be defined as a file scoped const. That way you're not redefining a function on every execution.

Reply:
That's good to know, I helped write it this way before it accepted an object in v6.

Reply:
Brian helped me make some updates to the PR to address this. Let me know if there's anything further you think I should do!